recipes/numpy_indexed/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version = "0.3.2" %}
+
+package:
+  name: numpy-indexed
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/EelcoHoogendoorn/Numpy_arraysetops_EP.git
+  git_tag: {{ version }}
+  
+build:
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+  number: 0
+  
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy
+    - pyyaml
+    - future
+  run:
+    - python
+    - numpy
+    - future
+
+test:
+  commands:
+    - py.test --pyargs numpy_indexed
+  imports:
+    - numpy_indexed
+  requires:
+    - pytest
+  
+about:
+  home: https://github.com/EelcoHoogendoorn/Numpy_arraysetops_EP
+  license: Freely Distributable
+  summary: Numpy extensions for set operations on nd-arrays, group_by operations,
+    and related functionality
+
+extra:
+  recipe-maintainers:
+    - ivoflipse
+    - Korijn
+    - EelcoHoogendoorn
```

Comment:
@EelcoHoogendoorn, are you ok with being added as a maintainer of `numpy-indexed` at [conda-forge]( http://conda-forge.github.io/ )?